open-metadata-implementation/user-interfaces/ui-admin-services/docs/user/README.md

```diff
@@ -0,0 +1,112 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+# OMAG Server Platform Administration Services User Guide
```

Comment:
Shouldn't this be UI specific now?But I think this is work in progress - so will skip over these

Reply:
fixed